### PR TITLE
Javascript tests engine

### PR DESCRIPTION
**Pull request by tomasperezv**
This PR adds 'es6' via babel + mocha support to the repo. 

I've included a module from the companies repository related with the unit tests framework and also implemented tests for it (tests for a unit tests framework, inception FTW).

### CODE REVIEW

**Comment by tomasperezv** [comment]
@jobandtalent/frontend

**Comment by Ivlo** [comment]
:ok_hand:

File: src/javascript/test-framework/mock.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+'use strict'
```
Comment (alejandroweb) [inline src/javascript/test-framework/mock.js]: Missing semicolon here

File: package.json [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+    "css-loader": "^0.23.1",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "less": "^2.6.1",
+    "less-loader": "^2.2.3",
+    "postcss-loader": "^0.8.2",
+    "style-loader": "^0.13.1",
+    "webpack": "^1.12.14"
+  },
+  "dependencies": {
+    "babel-preset-es2015": "^6.6.0",
```
Comment (alejandroweb) [inline package.json]: I think babel should go in devDependencies we use it while developing but is just an opinion :)

File: src/stylesheets/colors.less [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+@ultra-low-warning: rgba(255, 168, 0, 0.08);
+
+// Negative/Destructive
+@high-negative: #E04747;
+@negative: rgba(229, 83, 83, 1);
+@middle-negative: rgba(229, 83, 83, 0.6);
+@low-negative: rgba(229, 83, 83, 0.4);
+@ultra-low-negative: rgba(229, 83, 83, 0.08);
+
+@benefit-title: rgb(93, 109, 121);
```
Comment (alejandroweb) [inline src/stylesheets/colors.less]: I would remove benefit colors or I would change the name for something less specific.

Comment (tomasperezv) [inline src/stylesheets/colors.less]: Same than in the other PR, we'll change that in another PR.

File: package.json (hunk 2 above)
Comment (tomasperezv) [inline package.json]: I'm not 100% sure yet about how it's going to be used, so I'm keeping babel on normal dependencies, however, let's see how it evolves.

**Pull request merged**